src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Windows.targets

```diff
@@ -102,4 +102,12 @@ Copyright (c) .NET Foundation. All rights reserved.
     </ItemGroup>
 
   </Target>
+
+  <!-- Emit a warning if 'UseUwp' is set (it is only supported by the .NET 9 SDK) -->
+  <Target Name="_WarnForUnsupportedUseUwpFeatureEnabled"
+          AfterTargets="ResolvePackageAssets"
+          Condition=" '$(IncludeWindowsSDKRefFrameworkReferences)' == 'true'
```

Comment:
Can useuwp ever end up true if the include property is not true?

Reply:
Technically speaking there's nothing preventing someone from setting that when not targeting `-windows`, but that scenario is not supported and it doesn't do anything with the .NET 9 SDK either. For this target I basically copied the same conditions and order as the one we added in #https://github.com/dotnet/sdk/pull/41936 to add the new preview warning when enabling the feature. I thought keeping them similar would make sense? Let me know! 🙂